gsoc2012.dd

```diff
@@ -30,8 +30,7 @@ $(P Current mentors: Walter Bright (Digital Mars), Andrei Alexandrescu
 (Facebook), David Simcha (Johns Hopkins University), Alex R&oslash;nne
 Petersen (MCI), Adam Wilson (Prospective Software), Jens Mueller
 (Friedrich Schiller University, Jena) Jonas Drewsen (Unity
-Technologies), and $(LINK2 http://www.russel.org.uk/, Russel Winder)
-(Independent Consultant).)
+Technologies), and Russel Winder (Independent Consultant).)
```

Comment:
Google still lists russel.org.uk. Maybe it's just a temporary blackout. @russel?